specification/eventgrid/data-plane/Microsoft.ContainerService/stable/2018-01-01/ContainerService.json

```diff
@@ -27,6 +27,104 @@
           "description": "The highest PATCH Kubernetes version considered preview for the ManagedCluster resource. There might not be any version in preview at the time of publishing the event",
           "type": "string"
         }
+      },
+      "x-ms-examples": {
+        "newKubernetesVersionAvailableCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/new_kubernetes_version_available.json"
+        },
+        "newKubernetesVersionAvailableEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/new_kubernetes_version_available.json"
+        }
+      }
+    },
+    "ContainerServiceClusterSupportEndedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerService.ClusterSupportEnded event",
+      "type": "object",
+      "properties": {
+        "kubernetesVersion": {
+          "description": "The Kubernetes version of the ManagedCluster resource",
+          "type": "string"
+        }
+      },
+      "x-ms-examples": {
+        "clusterSupportEndedCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/cluster_support_ended.json"
+        },
+        "clusterSupportEndedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/cluster_support_ended.json"
+        }
+      }
+    },
+    "ContainerServiceClusterSupportEndingEventData": {
```

Comment:
Can we add a base model that both definitions reference named `ContainerServiceClusterSupportBaseProperties`?